src/playdate/graphics.nim

```diff
@@ -390,7 +420,7 @@ proc setBitmapMask*(
 
 proc getBitmapMask*(this: LCDBitmap): LCDBitmap =
     privateAccess(PlaydateGraphics)
-    return LCDBitmap(resource: playdate.graphics.getBitmapMask(this.resource), free: false) # Who should manage this memory? Not clear. Not auto-managed right now.
+    return newUnmanagedBitmap(playdate.graphics.getBitmapMask(this.resource))
```

Comment:
> LCDBitmap* playdate->graphics->getBitmapMask(LCDBitmap* bitmap);
Gets a mask image for the given bitmap, or returns NULL if the bitmap doesn’t have a mask layer. The returned image points to bitmap's data, so drawing into the mask image affects the source bitmap directly. The caller takes ownership of the returned LCDBitmap and is responsible for freeing it when it’s no longer in use.

I'd say this means the returned bitmap is managed

Reply:
Interesting -- I'm going to have to test this

Reply:
You're absolutely right. Great catch